src/utils/types.ts

```diff
@@ -74,3 +74,44 @@ export type FooterResponse = {
   network: Network[];
   link: Link;
 };
+
+export type MentorshipPageProps = {
+  mentorship: MentorshipProgrammeData;
+  error: string | null;
+};
+
+export type MentorshipProgrammeData = {
+  page: PageProps;
+  mentorSection: MentorOrMenteeProps;
+  menteeSection: MentorOrMenteeProps;
+  feedback: FeedbackProps;
+};
+
+export type PageProps = {
+  title: string;
+  description: string;
+};
+
+export type LinkProps = {
+  label: string;
+  url: string;
+};
+
+export type MentorOrMenteeProps = {
+  title: string;
```

Comment:
Types are usually not called Props - just call it something meaningful so it's clear what it is 

Do we distinguish Mentor and Mentee at some point?

Reply:
Maybe we can change the this type when we do distinguish them in the future? For now they are the same in the API response. I've renamed these types in the mean time